components/sections/osmo-cta/backers-section.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+
+import { cn } from "@/lib/utils";
+import Image from "next/image";
+import { useState } from "react";
+
+const backers = [
+  {
+    imageUri: "/assets/backers/Paradigm.svg",
+    name: "Paradigm",
+  },
+  {
+    imageUri: "/assets/backers/BainCapital.svg",
+    name: "BainCapital",
+  },
+  {
+    imageUri: "/assets/backers/RibbitCapital.svg",
+    name: "RibbitCapital",
+  },
+  {
+    imageUri: "/assets/backers/ElectricCapital.svg",
+    name: "ElectricCapital",
+  },
+  {
+    imageUri: "/assets/backers/Grayscale.svg",
+    name: "Grayscale",
+  },
+  {
+    imageUri: "/assets/backers/Maven11.svg",
+    name: "Maven11",
+  },
+  {
+    imageUri: "/assets/backers/Anagram.svg",
+    name: "Anagram",
+  },
+  {
+    imageUri: "/assets/backers/EtherealVentures.svg",
+    name: "EtherealVentures",
+  },
+  {
+    imageUri: "/assets/backers/RobotoVentures.png",
+    name: "RobotoVentures",
+  },
+  {
+    imageUri: "/assets/backers/Nascent.svg",
+    name: "Nascent",
+  },
+  {
+    imageUri: "/assets/backers/FigmentCapital.svg",
+    name: "FigmentCapital",
+  },
+  {
+    imageUri: "/assets/backers/BCAP.svg",
+    name: "BCAP",
+  },
+  {
+    imageUri: "/assets/backers/Anatoly.png",
+    name: "Anatoly Yakovenko",
+  },
+  {
+    imageUri: "/assets/backers/Naval.png",
+    name: "Naval",
+  },
+  {
+    imageUri: "/assets/backers/Ethan.png",
+    name: "Ethan Buchman",
+  },
+  {
+    imageUri: "/assets/backers/Eric.png",
+    name: "Eric Schmidt",
+  },
+  {
+    imageUri: "/assets/backers/Hasu.png",
+    name: "Hasu",
+  },
+  {
+    imageUri: "/assets/backers/Raj.png",
+    name: "Raj Gokal",
+  },
+];
+
+export default function BackersSection() {
+  const [isExpanded, setIsExpanded] = useState(false);
```

Comment:
If let the button behave like a checkbox, we can remove this react state and use "has" selector:
https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-descendants

So we can create an expandable list without any react state